addons/udes_stock/models/procurement.py

```diff
@@ -18,6 +18,17 @@ def _get_orderpoint_domain(self, company_id=False):
             domain += [('id', 'in', orderpoints)]
         return domain
 
+    @api.model
+    def _get_or_create_procurement_group(self, procurement_name):
```

Comment:
Can we use `mixin.stock.model` and then we will get get or create from there?

Reply:
Pushed the suggested change